code/datums/diseases/advance/symptoms/disfiguration.dm

```diff
@@ -0,0 +1,49 @@
+/*
+//////////////////////////////////////
+Disfiguration
+
+	Hidden.
+	No change to resistance.
+	Increases stage speed.
+	Slightly increases transmittability.
+	Critical Level.
+
+BONUS
+	Adds disfiguration trait making the mob appear as "Unknown" to others.
+
+//////////////////////////////////////
+*/
+
+/datum/symptom/disfiguration
+
+	name = "Disfiguration"
+	desc = "The virus liquefies facial muscles, disfiguring the host."
+	stealth = 2
+	resistance = 0
+	stage_speed = 3
+	transmittable = 1
+	level = 5
+	severity = 1
+	symptom_delay_min = 25
+	symptom_delay_max = 75
+
+/datum/symptom/disfiguration/Activate(datum/disease/advance/A)
+	. = ..()
+	if(!.)
+		return
+	var/mob/living/M = A.affected_mob
+	if (HAS_TRAIT(M, TRAIT_DISFIGURED))
+		return
+	switch(A.stage)
+		if(5)
+			ADD_TRAIT(M, TRAIT_DISFIGURED, DISEASE_TRAIT)
+		else
```

Comment:
might be better off to remove this so you can get the message when the trait is added.

Reply:
added a message here